executor/batch_point_get.go

```diff
@@ -153,6 +163,9 @@ func (e *BatchPointGetExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	}
 	for !req.IsFull() && e.index < len(e.values) {
 		handle, val := e.handles[e.index], e.values[e.index]
+		if e.memTracker != nil {
+			e.memTracker.Consume(int64(len(val)))
```

Comment:
Memory consuming of `val` happens since returned from `batchGetter.BatchGet()` in `BatchPointGetExec.initialize()`. I think it would be more accurate to track there.

Besides, memory usage decreases when `e.values` (can be) released. So we need to consume a "negative" usage in `BatchPointGetExec.Close()`.

Moreover, `req` consumes significant memory too. I think its memory usage should be tracked either.

Reply:
done